custom/icds_reports/tasks.py

```diff
@@ -1176,3 +1176,35 @@ def _bust_awc_cache():
     for key in reach_keys:
         cache.delete(key)
     create_datadog_event('redis: delete dashboard keys', 'finish')
+
+
+@task(queue='dashboard_comparison_queue')
+def run_citus_experiment_raw_sql(sql, data_source="Unknown"):
+    experiment_context = {
+        "data_source": data_source,
+        "sql_query": sql,
+    }
+    experiment = DashboardQueryExperiment(name="Dashboard Query Experiment", context=experiment_context)
+    with experiment.control() as control:
+        db_alias = get_icds_ucr_db_alias()
```

Comment:
Doesn't this already get executed in `SqlData` itself, why not capture results from that instead of doing a repeat calculation?

Reply:
answered https://github.com/dimagi/commcare-hq/pull/24098#discussion_r277877643